PublicationsRouter/v2/oai-pmh/VERBS.md

```diff
@@ -56,13 +55,24 @@ Returned information:
 
 * Metadata Format: oai_dc (schema: http://www.openarchives.org/OAI/2.0/oai_dc.xsd, namespace: http://www.openarchives.org/OAI/2.0/oai_dc/)
 
-PubRouter API request: 
+Example request: `GET https://pubrouter.jisc.ac.uk/oaipmh/repo/123456789?verb=ListMetadataFormats`
+
+Example return: 
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<OAI-PMH xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.openarchives.org/OAI/2.0/" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+    <responseDate>2018-05-23T14:21:04Z</responseDate>
+    <request verb="ListMetadataFormats">http://pubrouter.jisc.ac.uk/repo/012b7a4352904fb99ec5fc5e09ab0a0c</request>
```

Comment:
Is that a real ID in the URL? Suggest anyway replacing it by `123456789` as done above.